lib/ansible/modules/windows/win_dhcp_lease.ps1

```diff
@@ -0,0 +1,473 @@
+#!powershell
+
+# Copyright: (c) 2019 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$ErrorActionPreference = 'Stop'
```

Comment:
Not required, we set this for all module invocations in the Ansible module runner.